src/Settings/index.ts

```diff
@@ -16,3 +16,11 @@ export interface IPluginSettings {
   runAllTestsFirst?: boolean
   showCoverageOnLoad: boolean
 }
+
+export function isDefaultPathToJest(str) {
+  return str === null
+}
+
+export function isNotDefaultPathToJest(str) {
+  return !isDefaultPathToJest(str)
+}

```

Comment:
I've extracted the trivial function and it's complement to join the settings interface.